responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -159,6 +159,14 @@ def _add_counterfactual_config(self, new_counterfactual_config):
                     'The desired_range should not be None'
                     ' for regression scenarios.')
 
+        if new_counterfactual_config.feature_importance and\
```

Comment:
maybe instead of using \ you can break it into two variables:
```
has_feat_imps = new_counterfactual_config.feature_importance
total_CFs_check = new_counterfactual_config.total_CFs < 10
if has_feat_imps  and total_CFs_check:
  ...
```

Reply:
would probably keep the check the same.